dipper/sources/GWASCatalog.py

```diff
@@ -105,18 +108,17 @@ def process_catalog(self, limit=None):
         """
         raw = '/'.join((self.rawdir, self.files['catalog']['file']))
         LOG.info("Processing Data from %s", raw)
-        efo_ontology = RDFGraph(False, "EFO")
-        LOG.info("Loading EFO ontology in separate rdf graph")
-        efo_ontology.parse(self.files['efo']['url'], format='xml')
-        efo_ontology.bind_all_namespaces()
-        LOG.info("Finished loading EFO ontology")
 
         so_ontology = RDFGraph(False, "SO")
         LOG.info("Loading SO ontology in separate rdf graph")
         so_ontology.parse(self.files['so']['url'], format='xml')
         so_ontology.bind_all_namespaces()
         LOG.info("Finished loading SO ontology")
 
+        mondo_file = '/'.join((self.rawdir, self.files['mondo']['file']))
+        with open(mondo_file, 'r') as mondo_fh:
+            mondo_data = json.load(mondo_fh)
```

Comment:
safe_load

Reply:
json does not contain a safe_load

out of curiosity lead me to https://stackoverflow.com/questions/38813298/is-json-loads-vulnerable-to-arbitrary-code-execution